apps/nuxt3-ssr/components/MainNavigation.vue

```diff
@@ -10,7 +10,7 @@ interface PropType {
 }
 
 const props: PropType = withDefaults(defineProps<PropType>(), {
-  maximumButtonShown: 3,
```

Comment:
I am not sure where config lists, but I would menu to be 'cohorts', 'networks', 'variables'

<img width="410" alt="image" src="https://github.com/molgenis/molgenis-emx2/assets/120060/912c0afb-e2c6-46e3-815d-f36742c77d0c">